SpatialGDK/Source/SpatialGDKServices/Private/SSpatialOutputLog.cpp

```diff
@@ -274,37 +270,45 @@ void SSpatialOutputLog::FormatAndPrintRawErrorLine(const FString& LogLine)
 {
 	const FRegexPattern ErrorPattern = FRegexPattern(TEXT("level=(.*) msg=(.*) code=(.*) code_string=(.*) error=(.*) stack=(.*)"));
 	FRegexMatcher ErrorMatcher(ErrorPattern, LogLine);
+	FString LogMessage;
+	FString LogCategory = TEXT("SpatialService");
 
 	if (!ErrorMatcher.FindNext())
 	{
 		UE_LOG(LogSpatialOutputLog, Error, TEXT("Failed to parse log line: %s"), *LogLine);
-		return;
-	}
-
-	FString ErrorLevelText = ErrorMatcher.GetCaptureGroup(1);
-	FString Message = ErrorMatcher.GetCaptureGroup(2);
-	FString ErrorCode = ErrorMatcher.GetCaptureGroup(3);
-	FString ErrorCodeString = ErrorMatcher.GetCaptureGroup(4);
-	FString ErrorMessage = ErrorMatcher.GetCaptureGroup(5);
-	FString Stack = ErrorMatcher.GetCaptureGroup(6);
-
-	// The stack message comes with double escaped characters.
-	Stack = Stack.ReplaceEscapedCharWithChar();
 
-	// Format the log message to be easy to read.
-	FString LogMessage = FString::Printf(TEXT("%s \n Code: %s \n Code String: %s \n Error: %s \n Stack: %s"), *Message, *ErrorCode,
-										 *ErrorCodeString, *ErrorMessage, *Stack);
+		LogMessage = *LogLine;
```

Comment:
What's the purpose of this?

Reply:
if it can't be matched by the regex, then the message is the whole line